ComponentKit/Components/CKButtonComponent.h

```diff
@@ -33,7 +33,7 @@ struct CKButtonComponentAccessibilityConfiguration {
                     titleFont:(UIFont *)titleFont
                      selected:(BOOL)selected
                       enabled:(BOOL)enabled
-                       action:(CKComponentAction)action
+                       action:(const CKTypedComponentAction<UIEvent *> &)action
```

Comment:
Technically, passing this around by reference might be overkill. For very small structs like `CKTypedComponentAction`, it is probably almost as cheap to just copy everywhere instead of passing references.

It's your call — I would decide on a standard (always pass by reference, or always pass by value) and then let's enforce it, both for ComponentKit and internally at FB.